### ISSUE TITLE
Add LICENSE file to the repo

### ISSUE THREAD

**Issue by senthilkumarkj** [apache/airflow-client-go#18]
Currently the repo has no license file and [pkg.go.dev](https://pkg.go.dev/github.com/apache/airflow-client-go/airflow) isn't displaying the documentation for the package. Clarifying the license would also encourage the usage of the client. 

The individual files say the file is available under Apache 2.0 license, so I presume this is the license we'd want to add to the repo as well. 

(I've created a PR for this. I'm unsure how to notify, so creating an issue and linking it to resolve.)

**Comment by senthilkumarkj** [comment]
I apologize, as this did not fix the [issue](https://pkg.go.dev/github.com/apache/airflow-client-go/airflow). 

1) I apparently can not type LICENSE correctly the [first time](https://github.com/apache/airflow-client-go/pull/17#event-6158284218) and every time I correct it after the spellcheck points it out and I corrected in all the places except for the file name. 🤦🏽‍♂️ I apologize for not catching this earlier. :(

2) The airflow folder has the real go module and it should probably have the LICENSE file. I haven't worked with a module before with this format. So I've added the LICENSE file under airflow folder too. 

New PR #19. Again, I'm sorry I did not catch this earlier. Thank you!
